### ISSUE TITLE
try to make model in Class

### ISSUE THREAD

**Issue by gmajabokkum** [hunkim/DeepLearningZeroToAll#132]
I'm trying to make LSTM model in Class

but running with class I got Nan error 

can any one help?

here is my code

` class ForecastLSTM(object):

    def __init__(self, sequence_length, data_dim, output_dim, hidden_dim):

        self.X = tf.placeholder(tf.float32, [None, sequence_length, data_dim], name="X")
        self.Y = tf.placeholder(tf.float32, [None, output_dim ], name="Y")

        cell=tf.contrib.rnn.BasicLSTMCell(num_units=hidden_dim, state_is_tuple=True, activation=tf.tanh)
        outputs, _states= tf.nn.dynamic_rnn(cell, self.X, dtype=tf.float32)
        self.Y_pred = tf.contrib.layers.fully_connected(outputs[:,-1], output_dim, activation_fn=None)

        self.loss = tf.reduce_sum(tf.sqrt((self.Y_pred - self.Y)))

 with sess.as_default():

    forecast = ForecastLSTM(sequence_length=5,
                            data_dim=5,
                            output_dim=1,
                            hidden_dim=10)

    optimizer = tf.train.AdagradOptimizer(1e-2)
    train = optimizer.minimize(forecast.loss)

    sess.run(tf.global_variables_initializer())

    for i in range(10000):
        _, l = sess.run([train, forecast.loss], feed_dict={forecast.X: trainX, forecast.Y: trainY})
        print (i, l)

`
